packages/fast-components-styles-msft/src/checkbox/index.ts

```diff
@@ -0,0 +1,35 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles } from "@microsoft/fast-jss-manager";
+import { ICheckboxClassNameContract } from "@microsoft/fast-components-class-name-contracts-base";
+
+const styles: ComponentStyles<ICheckboxClassNameContract, IDesignSystem> = {
+    checkbox: {
+        display: "flex",
+        flexDirection: "row"
+    },
+    checkbox_input: {
+        cursor: "pointer",
+        width: "20px",
+        height: "20px"
+    },
+    checkbox_label: {
+        color: (config: IDesignSystem): string => {
+            return config.foregroundColor;
+        },
+        fontSize: "15",
```

Comment:
Let's use the type ramp here:

Import:
```
import { applyType } from "./utilties/typography";
```
Use:
```
 ...applyType("t7");
```

We may need to reset the lineHeight here as well with that, but this way if the ramp changes this will automatically update.